src/components/index.js

```diff
@@ -45,7 +45,7 @@ export {
   SaveExportDropdownFactory,
   PanelHeaderDropdownFactory
 } from './side-panel/panel-header';
-export {default as PanelHeaderAction} from './side-panel/panel-header-action';
+export {default as PanelHeaderActionFactory} from './side-panel/panel-header-action';
```

Comment:
Still need to export `PanelHeaderAction`

Reply:
Fixed